docs/index.md

```diff
@@ -2,77 +2,52 @@
 nav_order: 2
 ---
 
-# Getting started
+# Getting Started
 
-## Download the CLI
-
-UBUNTU20.04
-
-```bash
-wget https://datayoga.io/releases/dy_cli/latest/dy_cli-ubuntu20.04-latest.tar.gz -O /tmp/dy_cli.tar.gz
-```
-
-UBUNTU18.04
-
-```bash
-wget https://datayoga.io/releases/dy_cli/latest/dy_cli-ubuntu18.04-latest.tar.gz -O /tmp/dy_cli.tar.gz
-```
-
-RHEL8
+## Create Python Virtual Environment
 
 ```bash
-wget https://datayoga.io/releases/dy_cli/latest/dy_cli-rhel8-latest.tar.gz -O /tmp/dy_cli.tar.gz
+python -m venv venv
```

Comment:
change to 'python3'

Reply:
actually removed this section, so irrelevant anymore.